src/UCTNode.cpp

```diff
@@ -113,13 +113,16 @@ bool UCTNode::create_children(std::atomic<int>& nodecount,
         }
     }
 
-    link_nodelist(nodecount, nodelist, mem_full);
+    link_nodelist(nodecount, nodelist, min_psa_ratio);
+    m_is_expanding = false;
```

Comment:
I guess this could theoretically have been a source for race conditions, if it were not for the fact that this atomic store uses the strictest memory order, `memory_order_seq_cst`.